pycytominer/normalize.py

```diff
@@ -162,14 +162,26 @@ def normalize(
         fitted_scaler = scaler.fit(profiles.query(samples).loc[:, features])
 
     # Scale the feature dataframe
+    # See https://github.com/cytomining/pycytominer/issues/319
```

Comment:
Amazing to catch this Shantanu! Thanks for this proposed fix.

I wonder if a simpler solution is to output the numpy array from `spherize()` directly. i.e., the return line at https://github.com/shntnu/pycytominer/blob/33cbb30f8030fddc252d3f470358df289035f977/pycytominer/operations/transform.py#L165

should simply be `return X @ self.W` (or what you have edited to include, I haven't looked yet; basically, we should return the values instead of a DataFrame, which might be a simpler fix. Oh, but maybe this still breaks b/c the columns will be principal components and not `feature_df.columns`?

Reply:
Oh, I think I understand. We want to use `Spherize()` standalone. I wonder then if it is more pythonic to add an argument like `Spherize(return_numpy=False)` and then set `normalize()` internals to use `return_numpy=True` in `Spherize` internals. What do you think?

Reply:
What a great idea! This should work

Reply:
Works!